pkg/pillar/types/domainmgrtypes.go

```diff
@@ -324,6 +325,7 @@ type DiskConfig struct {
 }
 
 type DiskStatus struct {
+	VolumeKey    string
```

Comment:
You don't need this, right?

Reply:
We don't need this but keeping it to make the code similar.